sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java

```diff
@@ -863,6 +903,14 @@ private String getTrainingSasUri() {
         }
     }
 
+    private String getSelectionMarkTrainingSasUri() {
+        if (interceptorManager.isPlaybackMode()) {
+            return "https://isPlaybackmode";
+        } else {
+            return Configuration.getGlobalConfiguration().get(FORM_RECOGNIZER_SELECTION_MARK_BLOB_CONTAINER_SAS_URL);
+        }
+    }
```

Comment:
Don't need this, please use the `URL_TEST_FILE_FORMAT` to form the url by providing the filename and delete the Selection Mark file from the container. 
Look at the other examples.